public/docs/dart/latest/guide/template-syntax.jade

```diff
@@ -1,407 +1,125 @@
-include ../_util-fns
+extends ../../../ts/latest/guide/template-syntax.jade
 
-+includeShared('{ts}', 'intro')
+block includes
+  include ../_util-fns
+  - var _JavaScript = 'Dart';
+  - var __chaining_op = '<code>;</code>';
+  - var __new_op = '<code>new</code> or <code>const</code>';
+  - var mapApiRef = 'https://api.dartlang.org/stable/1.16.0/dart-core/Map-class.html';
+  - var __objectAsMap = '<b><a href="' + mapApiRef + '">Map</a></b>'
 
-:marked
-  The complete source code for the example app in this chapter is
-  [in GitHub](https://github.com/angular/angular.io/tree/master/public/docs/_examples/template-syntax/dart).
-
-+includeShared('{ts}', 'html-1')
-+makeExample('template-syntax/dart/lib/app_component.html', 'my-first-app')(format=".")
-+includeShared('{ts}', 'html-2')
-
-+includeShared('{ts}', 'interpolation-1')
-+makeExample('template-syntax/dart/lib/app_component.html', 'first-interpolation')(format=".")
-+includeShared('{ts}', 'interpolation-2')
-+makeExample('template-syntax/dart/lib/app_component.html', 'title+image')(format=".")
-+includeShared('{ts}', 'interpolation-3')
-+makeExample('template-syntax/dart/lib/app_component.html', 'sum-1')(format=".")
-+includeShared('{ts}', 'interpolation-4')
-+makeExample('template-syntax/dart/lib/app_component.html', 'sum-2')(format=".")
-+includeShared('{ts}', 'interpolation-5')
-
-+includeShared('{ts}', 'template-expressions-1')
-:marked
-  We write template expressions in a language that looks like Dart.
-  Many Dart expressions are legal template expressions, but not all.
-  Dart expressions that have or promote side effects are prohibited,
-  including:
-
-  * assignments (`=`, `+=`, `-=`, ...)
-  * creating instances using `new` or `const`
-  * increment and decrement (`++` and `--`)
-
-  Other notable differences from Dart syntax include:
-
-  * no support for Dart string interpolation; for example,
-    instead of `"'The title is $title'"`, you must use
-    `"'The title is ' + title"`
-  * no support for the bitwise operators `|` and `&`
-  * new [template expression operators](#expression-operators), such as `|`
-+includeShared('{ts}', 'template-expressions-context')
-+includeShared('{ts}', 'template-expressions-guidelines')
-:marked
-  Dependent values should not change during a single turn of the event loop.
-  If an idempotent expression returns a string or a number, it returns the same string or number
-  when called twice in a row. If the expression returns an object (including a `DateTime`, `Map`, or `List`),
```

Comment:
Actually, come to think of it, _everything_ is an object in Dart, so the parenthetical remark is actually potentially confusing for Dartisans.